site/docs/releases.md

```diff
@@ -72,6 +72,77 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
+### 1.8.0 release
+
+Apache Iceberg 1.8.0 was released on February 13, 2025.
+
+The 1.8.0 release contains bug fixes and new features. For full release notes visit [Github](https://github.com/apache/iceberg/releases/tag/apache-iceberg-1.8.0)
+* Deprecation / End of Support
+  - Spark 3.3
```

Comment:
I believe the indentation needs to have 2 more spaces here and everywhere else under 1.8.0 so that it's properly detected as a list item You might want to compare with the items we list under previous releases

Reply:
Good catch, it was rendering the same as earlier versions for me but we should still keep it consistent. Updated

Reply:
maybe something was fixed in a newer version when these are rendered, but I remember that we had to update all of the indentation so that this content is properly shown on the website